manifest.toml

```diff
@@ -0,0 +1,122 @@
+packaging_format = 2
+
+id = "seafile"
+name = "Seafile"
+description.en = "Open Source Cloud Storage"
+description.fr = "Stockage Cloud Open Source"
+
+version = "9.0.9~ynh1"
+
+maintainers = ["Josué Tille"]
+
+[upstream]
+license = "AGPL-3.0,Apache-2.0,MIT,GPL-2.0"
+website = "https://www.seafile.com"
+demo = "https://demo.seafile.com"
+admindoc = "https://manual.seafile.com"
+code = "https://github.com/haiwen/seafile-server"
+cpe = "cpe:2.3:a:seafile:seafile"
+
+[integration]
+yunohost = ">= 11.2.3"
+architectures = ["amd64", "arm64", "armhf"]
+multi_instance = false
+ldap = true
+sso = true
+disk = "1G"
+ram.build = "1G"
+ram.runtime = "500M"
+
+[install]
+    [install.domain]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "domain"
+
+    [install.path]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "path"
+    default = "/seafile"
+
+    [install.server_name]
+    ask.en = "Choose a name (3 - 15 letters or digits)"
+    ask.fr = "Choisissez un nom (3 - 15 letters or digits)"
+    type = "string"
+    example = "Seafile"
+    default = "Seafile"
+
+    [install.admin]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "user"
+
+    [install.admin_password]
+    ask.en = "Enter a password for the administrator"
+    ask.fr = "Entrez un mot de passe pour l'administrateur"
+    type = "password"
+
+    [install.init_main_permission]
+    help.en = "If it's not public, everybody which want to access to any page of seafile need to be authenticated on the SSO. On the public mode anybody can access to the authentication page. The shared link will be olso accessible by anybody who has this link."
+    help.fr = "Si n'est pas publique, n'importe qui veux accéder à n'importe quelle page de seafile doit être authentifié dans le SSO. Dans le mode publique n'importe qui peut accéder à la page d'authentification de seafile. Les liens partagé seront aussi accessible par n'import qui qui à ce liens."
+    type = "group"
+    default = "visitors"
+
+[resources]
+    [resources.system_user]
+    allow_email = true
+    home = "/opt/yunohost/__APP__"
```

Comment:
Why not use the standard path `/var/www/seafile` ?

Reply:
Because following the [FHS Standard](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) there are really no reason to store binary files on `/var`. And at my point of view storing theses files in `/var/www` which is dedicated for web file there might be security issues.
And untils https://github.com/YunoHost/issues/issues/1936 is fixed there are no reason to change this as changing this is a source of bug on the migration.

Reply:
@Josue-T why would it change *anything* that a file is a binary ? As long as it's executable, a binary or a text file is **exactly the same thing**. The kernel sees the shebang or ELF header, loads it, executes it. So talking about binaries is not relevant.
You could talk about executables indeed, but there *are* executables in /var/www already. For example, the `occ` command from nextcloud.

Your link about FHS doesn't talk about binaries, nor about /var/www.

Now, you could actually argue that yunohost shouldn't even install anything in /var/www and I could eventually agree that we should install apps in /srv. But I don't see you arguing about that…

The sole reason of regrouping all apps in /var/www is readability.
A user should not have to *know* if one app is a binary or php app to find where is store its configuration file… So, yeah, maybe that's not the best directory we have but at least it's starting to standardize and users will find their files more easily.